go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/spacemeshos/ed25519 v0.0.0-20190530014421-e235766d15a1
 	github.com/spacemeshos/merkle-tree v0.0.0-20191028110812-1908c3126c82
 	github.com/spacemeshos/poet v0.1.0
-	github.com/spacemeshos/post v0.0.0-20191225190235-dfb8a5803e6d
+	github.com/spacemeshos/post v0.0.0-20200707150818-013318bab6f4
```

Comment:
This seems unrelated...

Reply:
oh, I just figured out why it is actually related (https://github.com/spacemeshos/post/pull/38)... 👍🏻

Reply:
Note to self: read existing comments before posting your own 😆